xml/Microsoft.Extensions.FileProviders/PhysicalFileProvider.xml

```diff
@@ -330,13 +331,13 @@
             </param>
         <summary>
           <para>Creates a <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" /> for the specified <paramref name="filter" />.</para>
-          <para>Globbing patterns are interpreted by <see cref="T:Microsoft.Extensions.FileSystemGlobbing.Matcher" />.</para>
+          <para>Globbing patterns are interpreted by <seealso cref="T:Microsoft.Extensions.FileSystemGlobbing.Matcher" />.</para>
         </summary>
         <returns>
             An <see cref="T:Microsoft.Extensions.Primitives.IChangeToken" /> that is notified when a file matching <paramref name="filter" /> is added,
             modified, or deleted. Returns a <see cref="T:Microsoft.Extensions.FileProviders.NullChangeToken" /> if <paramref name="filter" /> has invalid filter
             characters or if <paramref name="filter" /> is an absolute path or outside the root directory specified in the
-            constructor <see cref="M:Microsoft.Extensions.FileProviders.PhysicalFileProvider.#ctor(System.String)" />.
+            constructor <seealso cref="M:Microsoft.Extensions.FileProviders.PhysicalFileProvider.#ctor(System.String)" />.
```

Comment:
Did these seealso come from source? If yes, shouldn't they be failing in source too?

```suggestion
            constructor <see cref="M:Microsoft.Extensions.FileProviders.PhysicalFileProvider.#ctor(System.String)" />.
```

Reply:
Yes, I've fixed them in source here: https://github.com/dotnet/runtime/pull/113027/files. Not sure why the compiler doesn't flag it.